tsconfig.json

```diff
@@ -4,10 +4,12 @@
 		"declaration": true,
 		"importHelpers": true,
 		"module": "commonjs",
+		"lib": ["es2020"],
 		"strict": true,
 		"target": "es2019",
 		"esModuleInterop": true,
 		"noErrorTruncation": true,
+		"useUnknownInCatchVariables": false,
```

Comment:
Ultimately I think we should remove this but it might be worth some team discussion about the best way to handle types of catch variables.